testcourses/testcourse2/chapter/mychapter.xml

```diff
@@ -1,3 +1,4 @@
 <chapter display_name="chapter name">
   <sequential url_name="mysequential"/>
+  <sequential url_name="sequence2"/>
```

Comment:
What if we have something like this in a chapter olx? is it `FileDoesNotExist` or `PointerAlreadyPointedAt`? I think its expected to have a `url_name` so we should add a test case here for such a scenrio. 

`<sequential url_name="some-other-sequential-not-found" />`

Reply:
Can you expand a bit on this? I don't quite follow the scenario you're laying out.